docs/tutorials/01_getting_started.ipynb

```diff
@@ -434,6 +438,70 @@
     "results.rounding_results.samples"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Solve the problem exactly using the `CplexOptimizer`\n",
+    "\n",
+    "To evaluate the performance of QRAO in approximating the optimal solution, we can use CPLEX to solve for the exact optimal solution as follows:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 31,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "optimal function value: 18.0\n",
+       "optimal value: [1. 0. 1. 0. 1. 0.]\n",
+       "status: SUCCESS"
+      ]
+     },
+     "execution_count": 31,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "cplex_optimizer = CplexOptimizer()\n",
+    "cplex_results = cplex_optimizer.solve(problem)\n",
+    "exact_fval = cplex_results.fval\n",
+    "\n",
+    "cplex_results"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The approximation ratio (QRAO's approximate optimal function value divided by the exact optimal function value) tells us how closely QRAO approximated the optimal solution to the problem."
```

Comment:
Why was this marked as resolved?

Reply:
Actually, I am starting to come around to the original wording, since "approximation ratio" is a relatively standard term, used in the literature on Goemans-Williamson, for instance.